tests/Db/OldPasswordMapperTest.php

```diff
@@ -20,10 +21,8 @@
  *
  */
 
-namespace OCA\PasswordPolicy\Tests\Db;
+namespace OCA\PasswordPolicy\Db;

```

Comment:
namespace does not match the folder - autoloader will not find this file.
it will not be autploaded - but still does not follow the rest of the code we have all over the place

Reply:
autoloader never loads this file. phpunit does, Tests for a Class should reside in the same namespace, but finish with *Test, see https://github.com/owncloud/core/issues/30739#issue-303801425